src/layout/Dropdown/DropdownComponent.tsx

```diff
@@ -1,11 +1,11 @@
 import React, { useCallback } from 'react';
 
-import { Combobox } from '@digdir/designsystemet-react';
+import { EXPERIMENTAL_Suggestion, Label as DSLabel } from '@digdir/designsystemet-react';
```

Comment:
Syns det ble litt mye "experimental", men feel free to ignore 😄 er også i `MultipleSelectComponent`.
```suggestion
import { EXPERIMENTAL_Suggestion as Suggestion, Label as DSLabel } from '@digdir/designsystemet-react';
```